packages/statemanager/src/statefulVerkleStateManager.ts

```diff
@@ -88,6 +90,7 @@ export class StatefulVerkleStateManager implements StateManagerInterface {
     this._caches = opts.caches
     this.keccakFunction = opts.common?.customCrypto.keccak256 ?? keccak256
     this.verkleCrypto = opts.verkleCrypto
+    this.accessWitness = new AccessWitness({ verkleCrypto: this.verkleCrypto })
```

Comment:
Still wonder if (respectively tend to think that) the `accessWitness` is out of place here in the SM and should rather move over to the EVM. Not something to necessarily prioritize but maybe you can also give this some thought along the sideline.

This is such an EVM specific task, to track all the state accesses for the gas costs, also the `AccessWitness` class itself even does not have any dependencies within SM and the usage is solely done in the EVM code.

![grafik](https://github.com/user-attachments/assets/fc2cee6f-2a0d-4a26-807c-728ce8814dd4)

So maybe this is rather a pretty simple move-over.

I tend to think this will structurally benefit us mid-term if we do.